### ISSUE TITLE
Spurious console message when uploadInBackground: false: "Upload in background can be enabled only when cropper is disabled"

### ISSUE THREAD

**Issue by keunwoo** [filestack/filestack-js#256]
Steps to reproduce:

- Create a picker with the following code using filestack-js 3.3.5:
```
    const picker = client.picker({
        fromSources: ['local_file_system'],
        uploadInBackground: false,
    });
    picker.open();
```
- Click the "pick" button and pick any image file. (It is not necessary to transform or upload this file.)

Expected result: browser console logs are silent since no error has occurred.

Actual result: The following message is logged on the browser console at warning severity:

```
Upload in background can be enabled only when cropper is disabled
```

This is noise which interferes with debugging other issues and complicates communication with users. It should be not be logged when the caller explicitly passes `uploadInBackground: false`.

I have a complete self-contained example in this tree:

https://github.com/keunwoo/filestack-repro-harness/tree/keunwoo-uploadInBackground

See README.md for the (simple) instructions on running this.

**Comment by pcholuj** [comment]
Hi. 
We will fix it in upcoming releases.

**Comment by pcholuj** [comment]
Fixed in v3.5.0
